letsencrypt/templates/nginx/letsencrypt

```diff
@@ -1,6 +1,12 @@
+{%- if "letsencrypt" in nginx_default_sites -%}
```

Comment:
Roles in this repo shouldn't reference variables from other roles. This variable introduces an implicit dependency on the `nginx` role in the `configuration` repo.

Instead of this approach, I'd recommend adding a boolean flag to this role to optionally inject `default_server` in the server block below.

Reply:
@mrgnr I agree with you, but I'm not sure how to avoid that implicit dependency since at the same time, we need to also disable `default_server` in the default LMS nginx config for HTTP. If I add the boolean flag here and have to use it in the LMS config, I also have an implicit dependency, just the other way around. Is there any smart way to solve this?